src/amo/commands/updateTaskbar.ts

```diff
@@ -17,38 +20,44 @@ export async function updateTaskbar() {
     return;
   }
 
-  const path = doc.uri.fsPath;
+  const filePath = doc.uri.fsPath;
   const rootFolder = vscode.workspace.workspaceFolders?.[0].uri.fsPath;
   if (!rootFolder) {
     return;
   }
-  const relativePath = path.replace(rootFolder, "");
-  const pathParts = relativePath.split("/");
-  let guid = pathParts[1];
-  const version = pathParts[2];
 
+  const relativePath = filePath.replace(rootFolder, "");
```

Comment:
So i actually think that splicing from the end won't work all of the time since there can be many more than 2 subfolders. If the path is `x/x/x/ ... /x/guid/version/x/x/ ... /x/file and we want the guid and version, i have to remove what isn't the relative path to the workspace so that i know the first 2 folders are guid and version and there are some arbitrary amount proceeding. 

This is still probably the wrong way of going about this, since this requires the addon folder to have a parent of the workspace root, but that's not necessarily true (could be `workspaceroot/reviews/guid/...`). Caching the review url and simply checking for the guid in the filepath when changing editors will work without these cases. 

If there's some persistent environment where the extension runs, I can keep a file there for the cache. Otherwise, the best thing I can think of is to do the same in the root folder of the workspace. It'll probably be necessary in the future anyways